modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc

```diff
@@ -0,0 +1,79 @@
+= Converters and serialization 
+:description: Connectors Converters and serialization
+:page-cloud: true
+
+== Key and value converters
+
+Each Redpanda message is a key and value record. Converters handle serialization and deserialization of data between Redpanda topic and a connector. Record key and value converters are configured separately. It can be specified using `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
+
+[NOTE]
+====
+If an external system requires structured data (like BigQuery, Snowflake, SQL database), you need to provide data with schema. Then you need to use Avro Converter, Protobuf Converter or JSON Converter with schema.
+====
+
+== ByteArray Converter
+
+The most primitive and high-throughput converter. Schema is ignored.
+It is a default converter type in Managed Connectors. 
+
+To use the converter, select `ByteArray` option as a key or value message format.
+
+== String Converter
+
+It is a high-throughput converter. Schema is ignored. All data are converted to a String.
+
+To use the converter, select `String` option as a key or value message format.
+
+== JSON Converter
+
+JSON Converter can be specified by selecting `JSON` option as a key or value message format.
+The connector needs a message schema to check message format. Usually, the schema is stored in Schema Registry. 
+
+However, if the Schema Registry cannot be used, embedded JSON schema is an option. Then each message contains a schema. It will result in a large size of a message. Example JSON message with embedded schema:
+[source,json]
```

Comment:
Can we make it more clear that for JSON Converter we only support schema embedded in the message?
"Usually, the schema is stored in Schema Registry" and "embedded JSON schema is an option" may be misleading.